### PR TITLE
Update service resource metrics and related code

### PR DESCRIPTION
**Pull request by anukul**
- Turn usage_recorder thread back on
- Update metrics recording logic
- Add service-tag-uuid label to pods, remove b16-service-name label

### CODE REVIEW

**Comment by anukul** [comment]
Fixing conflicts.

**Comment by judepereira** [comment]
@anukul also remove https://github.com/storyscript/platform-engine/pull/385/files#diff-2f0d1b4d2606154cee6668326a8da6c8R165 added during our debugging session.

**Comment by judepereira** [comment]
@anukul can you test your branch on staging?

**Comment by judepereira** [comment]
@anukul ready for review? does it work on staging?

**Comment by anukul** [comment]
Yes

File: asyncy/ServiceUsage.py [hunk 1]
```diff
@@ -76,99 +60,84 @@ def cpu_units(cls, value: str) -> float:
...
+                          logger: Logger):
+        cpu_units = []
+        memory_bytes = []
 
-        # Get metrics
         prefix = Kubernetes._get_api_path_prefix('metrics')
-        labels = cls.get_service_labels(service)
         qs = urllib.parse.urlencode({
-            'labelSelector': f'b16-service-name in ({",".join(labels)})'
+            'labelSelector': f'service-uuid={service_uuid},image-tag={tag}'
```
Comment (judepereira) [inline asyncy/ServiceUsage.py]: @anukul can tags contain illegal characters?

Comment (anukul) [reply asyncy/ServiceUsage.py]: Updated.

**Pull request merged**